ui/client/components/toolbars/status/ProcessInfo.tsx

```diff
@@ -122,7 +128,13 @@ class ProcessInfo extends React.Component<OwnProps & StateProps, State> {
                   {icon}
                 </div>
                 <div className={"process-info-text"}>
-                  <div className={"process-name"}>{process.name}</div>
+                  {isRenamePending ?
+                    (
+                      <div className="process-name process-name-rename" title={process.name}>{nextId}*</div>
```

Comment:
What effect do we want to achieve with this "title"?

Reply:
wanted some place with "old" value. there is "*" postfix in renamed (not yet saved) value. I think it's nice feature to show this as hint and only old value is enough in my opinion